proto/one/enterprise/data/report_definition_json_v2.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+import "enterprise_data_table_query_parameter.proto";
+import "post_filtering.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
+
+enum ReportDefinitionCategory {
```

Comment:
All of our other enums are prefixed with Enum..., might be nice to continue that pattern.  Makes it obvious when you are dealing with a model vs an enum.

Reply:
Sure, I can rename the enum.